src/Benchmarks/Program.cs

```diff
@@ -26,11 +27,17 @@ public static void Main(string[] args)
 
             Console.WriteLine($"Current directory: {Directory.GetCurrentDirectory()}");
 
+            var config = new ConfigurationBuilder()
+                .AddCommandLine(args)
+                .AddEnvironmentVariables(prefix: "ASPNETCORE_")
```

Comment:
Do you have to add the variables explicitly? https://github.com/aspnet/Hosting/commit/0f872b8f73e34f8d6cb24ad20587368a31b01f52